charts/internal/seed-controlplane/charts/packet-cloud-controller-manager/templates/deployment.yaml

```diff
@@ -27,6 +27,7 @@ spec:
 {{ toYaml .Values.podAnnotations | indent 8 }}
 {{- end }}
     spec:
+      serviceAccountName: cloud-controller-manager
```

Comment:
Why is the cloud-controller-manager now using a service account? The service account is for the seed, but the CCM should act against the shoot.

Reply:
yes i think i mixed up things here. I need the csi controller to run against the seed for pv/pvc bounding otherwise the shoot etcd won't work